tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,88 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    Click Link    Resources
+    Sleep    5s
+   ${list_of_tile} =    Get List Of Ids Of Tiles
+   Verify Star Icons Are Clickable    ${list_of_tile}
+
+    FOR  ${index}  IN RANGE    27    48
+         Click Element    //*[@id="${list_of_tile}[${index}]"]/div[1]/span
+         ${clicked} =     Get Element Attribute    //*[@id="${list_of_tile}[${index}]"]/div[1]/span    class
```

Comment:
please try to make this more readable/self-explanatory.

Reply:
Done